tests/integration/ux.test.ts

```diff
@@ -125,15 +122,23 @@ describe("Rally Web Platform UX flows", function () {
     await driver.executeScript("document.getElementById('accept').style.position='relative'");
     await driver.executeScript("document.getElementById('accept').style.top='-5rem'");
 
+    // NOTE: Clicking accept and enroll opens the extension page in chrome marketplace
+    // and hence we must return focus to Rally website before proceeding with further testing
+
+    const windowHandle = await driver.getWindowHandle();
+
     await findAndAct(
       driver,
       By.xpath('//button[text()="Accept & Enroll"]'),
       (e) => e.click()
     );
 
-    await driver.executeScript("window.scrollTo(0, document.body.scrollHeight);")
+    // Switch back to Rally website
+    driver.switchTo().window(windowHandle);
```

Comment:
I think this should have `await`.

Even with `await`, I think this approach might introduce a race condition: if the marketplace window launches between line 137 and 139, the test will still get stuck on the marketplace window.

Selenium docs shows [a way to wait for the window to launch](https://www.selenium.dev/documentation/webdriver/browser/windows/) before switching:
```
//Wait for the new window or tab
await driver.wait(
    async () => (await driver.getAllWindowHandles()).length === 2,
    10000
  );
```